maintainer-policies-and-guidelines.md

```diff
@@ -0,0 +1,29 @@
+# Maintainer Policies and Guidelines
+
+This document aims to clarify hard policies that Maintainers have to follow as well as less-enforced guidelines that are recommended to be followed. The goal is to help Maintainers understand what is expected of them and what actions should be done to reduce the chances of conflict within the Maintainer team.
+
+## Policies
+
```

Comment:
You can add that PRs must not be merged without checking with a maintainer that requested changes and/or got assigned the PR (possibly self-assigned). The review should not be dismissed without first checking with the maintainer. This also serves as a way for maintainers to request others to wait for their review.